pkgs/servers/monitoring/grafana-dash-n-grab/default.nix

```diff
@@ -0,0 +1,36 @@
+{ buildGoModule, fetchFromGitHub, lib }:
+
+buildGoModule rec {
+  pname = "grafana-dash-n-grab";
+  version = "0.3.1";
+
+  src = fetchFromGitHub {
+    rev = "v${version}";
+    owner = "esnet";
+    repo = "gdg";
+    sha256 = "sha256-M4V4ybOizXCLxTTuS17M0y2tRmjQmTIBdXVbumzOoeA=";
+  };
+
+  vendorSha256 = "sha256-pXo80Tean5OkQ0Sv2+/RlRnAtaClwGP7tuDC2irsh+E=";
+
+  ldflags = [
+    "-s"
+    "-w"
+    "-X main.Version=${version}"
+    "-X github.com/esnet/gdg/version.GitCommit=${src.rev}"
+  ];
+
+  doCheck = false;
```

Comment:
The test suite tries to communicate with a running version of grafana locally... It fails if you don't have grafana running.

Reply:
Could you add this as a comment above this line?